presto-hive/src/test/java/io/prestosql/plugin/hive/TestHiveIntegrationSmokeTest.java

```diff
@@ -4599,7 +4594,7 @@ public void testInvalidColumnsAnalyzeTable()
     @Test
     public void testAnalyzeUnpartitionedTableWithColumnSubset()
     {
-        String tableName = "test_analyze_columns_unpartitioned_table";
+        String tableName = "test_analyze_empty_column_set";
```

Comment:
Why change here?

(Note that the table name matched (roughly) the test name)

Reply:
Meant to change the other one, thanks.